Update integration docs for LLMQ-IS (#50)
dashpay/docs: developers/integration.rst

This documentation is also available as a `PDF <https://github.com/dashpay/docs/raw/master/binary/integration/Dash_v0.13_IntegrationOverview.pdf>`__.

> `Dash Core <https://github.com/dashpay/dash/releases>`__ is a “fork” of

Reviewer: ```suggestion
`Dash Core <https://github.com/dashpay/dash>`__ is a “fork” of
```

> Bitcoin and shares many common functionalities. Key differences relate

Reviewer: ```suggestion
`Bitcoin Core <https://github.com/bitcoin/bitcoin>`__ and shares many common functionalities. Key differences relate
```

.. _integration-instantsend:

InstantSend
===========

This documentation is also available as a `PDF <https://github.com/dashpay/docs/raw/master/binary/integration/Dash_v0.13_InstantSend.pdf>`__.

InstantSend is a feature provided by the Dash network that allows for
zero-confirmation transactions to be safely accepted and re-spent. All
valid transactions are secured by the Dash network at the moment of
broadcast. Every secured transaction is included in the next block in
accordance with standard blockchain principles.

InstantSend is enabled by the Masternode Network which comprises
approximately 5,000 masternode servers. These nodes are differentiated
from standard nodes by having proven ownership of 1,000 Dash, making the
network `highly resistant to Sybil attacks <https://en.wikipedia.org/wiki/Sybil_attack>`__. 
Masternodes form `Long-Living Masternode Quorums (LLMQs) <https://github.com/dashpay/dips/blob/master/dip-0006.md>`__, 
which are responsible for providing instant certainty to the transaction
participants that the transaction inputs cannot be respent, and that the

> transaction will be included in the next block instead of a conflicting

Reviewer 2: There is a guarantee that a conflicting tx will not be mined, but there is _not_ a guarantee that locked txs will appear in the next block.

Reviewer 3: ```suggestion
transaction will be included in a following block instead of a conflicting
```
alright like this?

Broadcasting InstantSend Transactions
-------------------------------------

> Since Dash 0.14.0 established LLMQ on the Dash network, quorums will now

Reviewer 4: ```suggestion
Since Dash v0.14.0 established LLMQs on the Dash network, quorums will now
```

attempt to lock every valid transaction by default without any

> additional fee or action by the sending wallet or user. A transaction is

Reviewer 3: ```suggestion
additional fee or action by the sending wallet or user.
```

> eligible for InstantSend when each of its inputs is considered

Reviewer 3: ```suggestion
```

> confirmed. This is the case when: 

Reviewer: Should maybe clarify that it's `or` for conditions below i.e. you either one should be `true` for input to be considered confirmed.

Reviewer 3: ```suggestion
confirmed. This is the case when at least one of the following circumstances is true: 
```
